nextcord/ext/checks/core.py

```diff
@@ -0,0 +1,599 @@
+"""
+The MIT License (MIT)
+Copyright (c) 2021-present tag-epic
```

Comment:
Isnt this mostly copy pasted from ext.commands? If so it needs Dannys copyright header